.circleci/config.yml

```diff
@@ -14,22 +14,11 @@ jobs:
       - node/install-yarn:
           version: 1.22.10
 
-      - node/with-cache:
-          cache-key: yarn.lock
+      - node/install-packages:
+          with-cache: true
           cache-version: v3
           include-branch-in-cache-key: false
-          steps:
-            - run:
-                name: yarn install
-                command: |
-                    CKSUM_BEFORE=$(cksum yarn.lock)
-                    yarn install
-                    CKSUM_AFTER=$(cksum yarn.lock)
-                  
-                    if [[ $CKSUM_BEFORE != $CKSUM_AFTER ]]; then
```

Comment:
Is this done by the orb now? 🤔

Reply:
Yeah, the new version does this for you.